extension/communicate-to-arcs.js

```diff
@@ -5,21 +5,24 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
-function _getUrl(cdnRoot, arcManifest, amKey) {
-  return cdnRoot+'?manifest='+encodeURIComponent(arcManifest)+'&amkey='+amKey;
+function _getUrl(cdnRoot, arcManifest, response) {
+  let ret = `${cdnRoot}?manifest=${arcManifest}&amkey=${response.amKey}`;
```

Comment:
Do folks use something like https://medialize.github.io/URI.js/ to build safe urls in JS? How does Polymer handle this? Perhaps worth a TODO comment to look at sorting this out down the line.